src/components/Settings/General.vue

```diff
@@ -1,56 +1,129 @@
+      <Button class="p-button-raised mr-2" :label="$t(`components.setting.general.buttons.editlabel`)"
+        :disabled="saveDiabale" @click="saveOrderList()" ></Button>
+
     </div>
+    <OrderList v-model="listOfExperiences" listStyle="height:auto" stripedRows="true" dataKey="id"
+      @reorder="menuList($event.value, $event.originalEvent)">
+      <template #header>
+        {{
+          $t(`components.setting.general.experience`)
+        }}
+
+      </template>
+
+      <template #item="slotProps">
+        <div class="product-item">
+          <div class="image-container">
+            <label for="username">
+              {{ slotProps.item.label }}
+            </label>
+          </div>
+          <div class="product-list-detail">
+            <InputSwitch :model-value="slotProps.item.enabled" @input="handleFeatureToggle($event, slotProps.item.id)" />
+            <div v-tooltip="$t(`tooltips.edit-experience`)" class="icon-wrapper hover:text-primary"
+              @click="editDialogToggle(slotProps.item)">
+              <i class="pi pi-pencil"></i>
+            </div>
+          </div>
+        </div>
+      </template>
+    </OrderList>
+    <ExperiencePopup v-if="editModalIsOpen" :payload="editModalPayload" :type="type"
+      :fetching-latest-experience="fetchingLatestExperience" @close="editCloseModal" />
   </div>
 </template>
 <script>
-  import InputSwitch from "primevue/inputswitch"
-  import { getModule } from "vuex-module-decorators"
-  import Settings from "@/store/Modules/Settings"
-  const settingsModule = getModule(Settings)
-  export default {
-    name: "General",
-    components: { InputSwitch },
-    computed: {
-      experimentalFeatures() {
-        return settingsModule.data.experimentalFeatures
-      },
+import InputSwitch from "primevue/inputswitch"
+import { getModule } from "vuex-module-decorators"
+import Settings from "@/store/Modules/Settings"
+import Button from "primevue/button"
+import Dialog from "primevue/dialog"
+import { nanoid } from "nanoid"
+import OrderList from 'primevue/orderlist';
+
+import { errorToast, successToast } from "@/utils/toastService"
+import ExperiencePopup from "@/components/Dialog/ExperiencePopup.vue"
+const settingsModule = getModule(Settings)
+export default {
+  name: "General",
+  components: { InputSwitch, OrderList, Button, ExperiencePopup },
+  data() {
+    return {
+      listOfExperiences: [],
+      updatedExperienceList: [],
+      label: "",
+      icon: "",
+      url: "",
+      editModalPayload: {},
+      editModalIsOpen: false,
+      type: "",
+      saveDiabale: true
+    }
+  },
+
+  computed: {
+    editDialog() {
+      return settingsModule.data.editDialog
     },
-    methods: {
-      handleFeatureToggle(enabled, id) {
-        settingsModule.toggleExperimentalFeatures({ id, enabled })
-      },
+  },
+  mounted() {
+    this.listOfExperiences = JSON.parse(JSON.stringify(settingsModule.data.listOfMenu))
```

Comment:
you can use the function defined below for this.

Reply:
Done